libpod/image/prune.go

```diff
@@ -0,0 +1,35 @@
+package image
+
+import "fmt"
+
+// GetUnamedImages returns a slice of images that have no names/unused
+func (ir *Runtime) GetUnamedImages() ([]*Image, error) {
+	var (
+		unamedImages []*Image
+	)
+	allImages, err := ir.GetImages()
+	if err != nil {
+		return nil, err
+	}
+	for _, i := range allImages {
+		if len(i.Names()) == 0 {
+			unamedImages = append(unamedImages, i)
+		}
+	}
+	return unamedImages, nil
+}
+
+// Prune removes all unnamed and unused images from the local store
+func (ir *Runtime) Prune() error {
+	pruneImages, err := ir.GetUnamedImages()
+	if err != nil {
+		return err
+	}
+	for _, i := range pruneImages {
+		if err := i.Remove(true); err != nil {
+			return err
+		}
+		fmt.Println(i.ID())
```

Comment:
Shouldn’t output happen in `cmd/podman`?

Reply:
yes, great point.